dxm/src/main/java/io/pcp/parfait/dxm/PcpOffset.java

```diff
@@ -5,4 +5,5 @@
 
 interface PcpOffset {
```

Comment:
This classes role is now getting lost on me.  If an instance of this clase is to represent an offset+length, would `length` be a better name that `byteSize` ?